sbin/build.sh

```diff
@@ -263,22 +263,27 @@ removingUnnecessaryFiles()
 signRelease()
 {
   if [ "$SIGN" ]; then
-    if [[ "$OSTYPE" == "cygwin" ]]; then
-      echo "Signing release"
-      signToolPath=${signToolPath:-"/cygdrive/c/Program Files/Microsoft SDKs/Windows/v7.1/Bin/signtool.exe"}
-      # Sign .exe files
-      FILES=$(find "${OPENJDK_REPO_TAG}" -type f -name '*.exe')
-      for f in $FILES; do
-        "$signToolPath" sign /f "$CERTIFICATE" /p "$SIGN_PASSWORD" /fd SHA256 /t http://timestamp.verisign.com/scripts/timstamp.dll "$f"
-      done
-      # Sign .dll files
-      FILES=$(find "${OPENJDK_REPO_TAG}" -type f -name '*.dll')
-      for f in $FILES; do
-        "$signToolPath" sign /f "$CERTIFICATE" /p "$SIGN_PASSWORD" /fd SHA256 /t http://timestamp.verisign.com/scripts/timstamp.dll "$f"
-      done
-    else
-      echo "Skiping code signing as it's only supported on Windows"
-    fi
+    case "$OSTYPE" in
+      "cygwin")
+        echo "Signing Windows release"
+        signToolPath=${signToolPath:-"/cygdrive/c/Program Files/Microsoft SDKs/Windows/v7.1/Bin/signtool.exe"}
+        # Sign .exe files
+        FILES=$(find "${OPENJDK_REPO_TAG}" -type f -name '*.exe')
+        echo "$FILES" | while read -r f; do "$signToolPath" sign /f "$CERTIFICATE" /p "$SIGN_PASSWORD" /fd SHA256 /t http://timestamp.verisign.com/scripts/timstamp.dll "$f"; done
+        # Sign .dll files
+        FILES=$(find "${OPENJDK_REPO_TAG}" -type f -name '*.dll')
+        echo "$FILES" | while read -r f; do "$signToolPath" sign /f "$CERTIFICATE" /p "$SIGN_PASSWORD" /fd SHA256 /t http://timestamp.verisign.com/scripts/timstamp.dll "$f"; done
+      ;;
+      "darwin"*)
+        echo "Signing OSX release"
+        # Sign all executables
```

Comment:
So @neomatrix369 's comment probably got missed (https://github.com/AdoptOpenJDK/openjdk-build/pull/165#discussion_r139806299).

>is it possible to use a constant for +111 so we know what files and which types of permissions are being filtered?

I don't think there is a constant that can be used here, but I think changing the comment to:

```bash
# Sign all files with the executable permission bit set.
```

Might be clear enough.

Reply:
Using the permission symbolic names might also make it clearer. So `+u+x,g+x,o+x` and `/u+x,g+x,o+x` instead of `+111` and `/111`.